src/golang/lib/collections/integration/config.go

```diff
@@ -45,6 +45,12 @@ type LambdaIntegrationConfig struct {
 	RoleArn string `json:"role_arn" yaml:"roleArn"`
 }
 
+type DatabricksIntegrationConfig struct {
+	WorkspaceUrl         string `json:"workspace_url" yaml:"workspaceUrl"`
+	AccessToken          string `json:"access_token" yaml:"accessToken"`
+	S3InstanceProfileArn string `json:"s3_instance_profile_arn" yaml:"s3InstanceProfileArn"`
```

Comment:
I'd like us to get in the habit of commenting the fields of configuration structs. Can we add a few sentences for each? Can also copy the descriptions over to the job manager config.